eex/metadata/additional_metadata.py

```diff
@@ -138,12 +138,12 @@
     }
 }
 
-mixing_rules = {"lorentz_berthelot": "_lorentz_berthelot",
-                "arithmetic" : "",
-                "geometric": ["geometric"],
-                "kong": ["kong"],
-                "sixth_power": ["sixth_power"],
-                "custom": ["custom"]}
+mixing_rules = ["lorentz_berthelot",
```

Comment:
This list is not used and is duplicated in eex/nb_converter.py (this is dictionary where keys are the possible mixing functions). This structure should be removed to avoid confusion